internal/uvm/create_lcow.go

```diff
@@ -99,7 +99,11 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		opts.Owner = filepath.Base(os.Args[0])
 	}
 
-	// TODO: If > RS5 use KernelDirect. If rootfs.vhd exists use it.
+	if _, err := os.Stat(filepath.Join(opts.BootFilesPath, "rootfs.vhd")); err != nil {
```

Comment:
It looks like if there is a rootfs.vhd in C:\Program Files\Linux Containers, but the caller changes opts.BootFilesPath to some other path that only has an InitRD, the code will still try to use a rootfs.vhd that isn't there.

Not sure what we can realistically do about this issue, but may be worth calling out.

Reply:
This is only internally callers and not a single caller changes this location. Honestly we should just remove the parameter and hard code it IMO.

Reply:
I'd be fine with removing the parameter.

Reply:
It's a storage-opt in docker, but it's v1, so I'll remove that when we get it moved to containerd.